src/comments/entities/comment.entity.ts

```diff
@@ -0,0 +1,39 @@
+import { BlogEntity } from '../../blogs/model/blog.entity';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  Tree,
+  UpdateDateColumn,
+} from 'typeorm';
+
+@Entity('comments')
+export class CommentEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'varchar', length: 250 })
+  title: string;
+
+  @Column()
+  is_published: boolean;
+
```

Comment:
May be, by deauflt this should be `false`?
But also it might complicate the frontend if this functionalily going to be supported...
How am I going to redit a comment?, where in the UI, what if I have two draft comments in the same blog...etc 
In my opinion all comments should be published, no need for such complexity... 
Note: you can still save a draft of the comment in front-end using localstroage (which is by the way how github sometimes saves your draft)

Reply:
alright! I've just removed it :+1: